config/yurtctl-servant/setup_edgenode

```diff
@@ -132,42 +155,18 @@ setup_yurthub() {
     do
         sleep 10
         # NOTE: context variables need to be replaced before exeuction
-        local podPhase
-
-        if [ "$provider" == "ack" ]; then
-            # the ack's apiserver doesn't support anonymous user visit, use 
-            # kubectl to check the pod status
-            podPhase=$(kubectl get po/yurt-hub-__node_name__\
-                --kubeconfig $KUBELET_CONF -n kube-system |
-                awk 'NR>1{print $3}') 
-        else
-            local podStat=$(curl -s \
-                https://__kubernetes_service_host__:__kubernetes_service_port_https__\
-/api/v1/namespaces/kube-system/pods/yurt-hub-__node_name__ \
---cert /var/lib/kubelet/pki/kubelet-client-current.pem \
---key /var/lib/kubelet/pki/kubelet-client-current.pem \
---cacert ${MINIKUBE_PKI_DIR}/ca.crt)
-            # yurt-hub pod is not found
-            if [ -n "$(echo "$podStat" | grep NotFound)" ]; then                 
-                retry=$((retry-1))
-	            log "yurt-hub-$NODE_NAME is not found, will retry $retry times"
-                continue
-            fi
-            # yurt-hub pod is created, let's check if the phase is "running"
-            podPhase=$(echo "$podStat" | 
-                grep '\"phase\":' | 
-                awk -F ':' '{print $2}' | 
-                tr -d '", ')
-        fi
+        set +e
+        local hub_healthz
+        hub_healthz=$(netstat -nlutp | grep 10261 | wc -l)
```

Comment:
if netstat tool is not installed, yurthub status can not be checked correctly?

Reply:
if netstat tool is not installed, the script will exit.